client/cli/src/arg_enums.rs

```diff
@@ -224,7 +228,7 @@ impl std::str::FromStr for Database {
 impl Database {
 	/// Returns all the variants of this enum to be shown in the cli.
 	pub fn variants() -> &'static [&'static str] {
-		&["rocksdb", "paritydb-experimental", "auto"]
+		&["rocksdb", "paritydb", "paritydb-experimental", "auto"]
```

Comment:
I would just remove paritydb-experimental from the list (not sure if it is working).

Reply:
It does not work. Clap produces an error if an unknown variant is used.